modules/backup-etcd.adoc

```diff
@@ -43,6 +43,11 @@ sh-4.2# chroot /host
 
 . Run the `cluster-backup.sh` script and pass in the location to save the backup to.
 +
+[TIP]
+====
+The `cluster-backup.sh` script is maintained as a component of `cluster-etcd-operator` and is a wrapper around the `etcdctl snapshot save` command.
```

Comment:
It's referred to as the "etcd cluster Operator" elsewhere in the docs, so:

"component of `cluster-etcd-operator`" -> "component of the etcd Cluster Operator"